internal/statestore/redis_indices.go

```diff
@@ -58,6 +58,8 @@ func extractIndexedFields(cfg config.View, t *pb.Ticket) map[string]float64 {
 				value = 1
 			}
 			result[boolIndexName(attribute)] = value
+		case *structpb.Value_StringValue:
```

Comment:
Could you as a separate task document the usage of filters / expectations of what needs to be specified etc.? Some easy reference we can point folks to for the current indexing / filtering mechanism?

Reply:
Forgot to add strings to the documentation above.  Are you expecting more?

Reply:
I meant website documentation on usage - not so much the code documentation. Again, not for this PR - but something the user guide should have.